src/components/pager/pager.tsx

```diff
@@ -1,8 +1,10 @@
-import React from "react"
-import * as pagerStyles from "../css/pager.module.scss"
-import Pages from "../components/pages"
+import React from "react";
 
-import { Link } from "gatsby"
+import * as pagerStyles from "./pager.module.scss";
+
+import Pages from "../pages/pages";
+
+import { Link } from "gatsby";
```

Comment:
`import { Link } from "gatsby";` shoud be up the `import Pages from "../pages/pages";`

Reply:
Done